### PR TITLE
Reorg and Ros2 Humble

### PR DESCRIPTION
**Pull request by hilary-luo**
This is a major change to the structure of how to access information on the website and how the non-nav data is managed. 

### The major changes are as follows:
- **Reducing the number of tabs** at the top by combining all robot accessories into one entry under robots (includes manipulators, computers, sensors etc). Renaming Software to ROS to better reflect the contents. Both of these changes better reflect the layout of our marketing website.

- **Removing the tools tab** and redistributing the information
    - NTP and IP address info moved under networking under ROS
    - PACs listed as its own accessories page
    - Physical Tools and Colours moved to a common section under robots

- **Moving OutdoorNav and IndoorNav to the navbar** to make them easy to find and give emphasis as products

- **Added versioning drop downs**, 1 for robots + ROS and 1 for outdoorNav. These are responsive in that they only show up when you are browsing that section of the website. These default to the latest version the first time you access them and automatically update for new versions. OutdoorNav also has a legacy version listed which takes you to a page with links to the old documentation.

- **Left side bar only shows the relevant information** for the tab that you are browsing.

- **Robot & ROS sections are versioned by ROS version** (currently ROS 1 Noetic or ROS 2 Humble). Thus, ROS 2 information has been removed from the ROS 1 version and vice versa

- Filling the ROS 2 ROS tab with **documentation for the humble software release**

### Relevant notes:
- Some images and components are being versioned within the docs folders while images and components that are common across ROS 1 and ROS 2 documentation are still in the static and components folders respectively. In the future as we update hardware pages we could convert more of that into common components to prevent having to maintain mutliple copies of the same information.

- Links had to all be changed to relative links in order for the versioning to work properly (shortcuts were taken in the ROS 1 version for obsolete components - not deemed worth the effort to turn into relative links)

- "Next" version is now being hidden and has to be enabled if we want to use it for development. 
- Readme was updated to reflect the new processes

### CODE REVIEW

File: docs/ros/ros.mdx
Comment (jhiggins-cpr) [inline docs/ros/ros.mdx]: For each of these sensors, can we use the same name as what is used under docs/robots/accessories/sensors? Also, for each of these sensors, can we provide a hyperlink to the corresponding page under accessories?

File: docs/ros/ros.mdx
Comment (jhiggins-cpr) [inline docs/ros/ros.mdx:94]: 1. The drivers for these sensors all get installed as part of the Clearpath Robot metapackage, right? If so, can we add an info box here to note that and to provide a link to docs/ros/installation/robot#clearpath-robot-metapackage?
2. Can we also add a note or info box to indicate that while other sensors are compatible that the ones outlined in the table below are the ones that have been validated by Clearpath?

File: docusaurus.config.js
Comment (jhiggins-cpr) [inline docusaurus.config.js]: We need to change this back to "throw" before merging

File: docs/ros/tutorials/_category_.json
Comment (jhiggins-cpr) [inline docs/ros/tutorials/_category_.json:1]: Are we adding a tutorial for ROS 1 Noetic to ROS 2 Humble upgrades? Refer to https://docs.clearpathrobotics.com/docs/robots/outdoor_robots/husky/tutorials_husky/tutorials_husky_ros1#backing-up-robot-configuration and https://clearpathrobotics.com/assets/guides/melodic/melodic-to-noetic/index.html.

For the Client Success team, this will be the single most important part of the Humble documentation.

**Review by jhiggins-cpr — COMMENTED** [review]
Overall, it's looking great. I've added a few suggestions throughout.

File: docs/ros/ros.mdx
Comment (roni-kreinin) [reply docs/ros/ros.mdx]: Fixed in https://github.com/clearpathrobotics/cpr-documentation/pull/129

File: docusaurus.config.js
Comment (roni-kreinin) [reply docusaurus.config.js]: Fixed in https://github.com/clearpathrobotics/cpr-documentation/pull/129

File: docs/ros/tutorials/_category_.json
Comment (roni-kreinin) [reply docs/ros/tutorials/_category_.json:1]: Added to #128

**Comment by roni-kreinin** [comment]
LGTM

File: docs/robots/outdoor_robots/husky/tutorials_husky.mdx
Comment (jhiggins-cpr) [inline docs/robots/outdoor_robots/husky/tutorials_husky.mdx]: Change "ROS2" to "ROS 2" throughout all files

**Review by jhiggins-cpr — APPROVED** [review]
Just one change

File: docs/robots/outdoor_robots/husky/tutorials_husky.mdx
Comment (hilary-luo) [reply docs/robots/outdoor_robots/husky/tutorials_husky.mdx]: Done

**Review by jhiggins-cpr — APPROVED** [review]

**Comment by rhysfaultless-cpr** [comment]
@hilary-luo please squash-and-merge this one when you are ready.
Squashing will simplify the rebase process for other people.
I do not intend to review this since @jhiggins-cpr already has.

**Pull request merged**
